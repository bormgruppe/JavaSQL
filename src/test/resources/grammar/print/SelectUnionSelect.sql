select 11 union
select 2
