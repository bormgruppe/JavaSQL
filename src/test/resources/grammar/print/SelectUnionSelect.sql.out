sql
	statement
		select
			value
				primary
					numeric: 11
	union
	statement
		select
			value
				primary
					numeric: 2
